blueman/plugins/AppletPlugin.py

```diff
@@ -26,28 +24,38 @@ def __init__(self, parent):
 
         self.__overrides = []
 
+        self._dbus_service = parent.DbusSvc
+        self._dbus_methods = set()
+        self._dbus_signals = set()
```

Comment:
Using ``set()``  is likely to cause a problem when adding a method/signal name more than once. On export it will result in an error so we better handle this. I suggest check for already existing methods/signals in ``DbusService``. And  in ``AppletPlugin`` catch the error from ``DbusService`` and raise ``ImportError`` for ``PluginManager``.